sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuMultiFileReader.scala

```diff
@@ -1406,8 +1406,7 @@ abstract class MultiFileCoalescingPartitionReaderBase(
       logDebug(s"$getFileFormatShortName Coalescing reading estimates the initTotalSize:" +
         s" $initTotalSize, and the true size: $finalBufferSize")
       SpillableHostBuffer(finalBuffer,
-        finalBufferSize,
-        SpillPriorities.ACTIVE_BATCHING_PRIORITY)
```

Comment:
I know we don't use spill priorities, but I would like us to not have changes unrelated to shuffle in this PR. Please, can we add this back? I'd like us to look at spill priorities later as we figure out how to actually prioritize, and want to leave user intent in the code.